onnx_chainer/functions/math.py

```diff
@@ -274,3 +274,13 @@ def gensym():
     o = object()
     dummy_objects.append(o)
     return str(id(o))
+
+
+def convert_Square(func, onnx_op_name, opset_version, input_names,
+                   output_names, parameters):
+    # 'Pow' operator's exponent accepts only floating value
+    exponent = chainer.Parameter(np.array(2, dtype=np.float32))
+    parameters.append(exponent)
+    input_names.append(str(id(exponent)))
+    if opset_version == 7:
+        return helper.make_node(onnx_op_name, input_names, output_names),
```

Comment:
I'd use Mul(x, x) instead of Pow(x, 2) as I believe `Pow` is usually slower than `Mul`.

Reply:
source code looks more simple, thx